src/Quadrature/quadrature.jl

```diff
@@ -209,21 +209,28 @@ end
 function FaceQuadratureRule{RefTriangle, T}(quad_type::Symbol, order::Int) where T
     qr = QuadratureRule{RefLine, T}(quad_type, order)
     # Shift interval from (-1,1) to (0,1)
-    for i in eachindex(qr.weights, qr.points)
-        qr.weights[i] /= 2
-        qr.points[i] = (qr.points[i] + Vec{1,T}((1,))) / 2
-    end
-    return create_face_quad_rule(RefTriangle, qr.weights, qr.points)
+    # for i in eachindex(qr.weights, qr.points)
+    #     qr.weights[i] /= 2
+    #     qr.points[i] = (qr.points[i] + Vec{1,T}((1,))) / 2
+    # end
```

Comment:
Perhaps a short note for each if/how the intervals are shifted? (And remove the old code)

Reply:
I need to stop forgetting commented stuff :"D